rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/util/EvaluatorUtil.java

```diff
@@ -310,9 +310,19 @@ public static CompletableFuture<Evaluator> makeFutureEvaluator(ExecutorService e
             String jobName = "Loading " + label;
             try {
                 services.jobStart(jobName, imports.length);
-                Evaluator eval = ShellEvaluatorFactory.getDefaultEvaluator(Reader.nullReader(),
-                        IoBuilder.forLogger(customLog).setLevel(Level.INFO).buildPrintWriter(),
-                        IoBuilder.forLogger(customLog).setLevel(Level.ERROR).buildPrintWriter(), services);
+                Evaluator eval;
+                Reader nullReader = Reader.nullReader();
+                PrintWriter infoWriter = IoBuilder.forLogger(customLog).setLevel(Level.INFO).buildPrintWriter();
```

Comment:
nit: a better name would be `var out` and `var err`. since that is what we're building, the stdout & stderr writers for the evaluator.